### DISCUSSION TITLE
Reporting add-on

### DISCUSSION THREAD

**Discussion by remcotolsma** [pronamic/wp-pronamic-pay#122] [Ideas]
## Links

- https://woocommerce.com/product-category/woocommerce-extensions/marketing-extensions/reporting-extensions/

## Internal Links

- **Basecamp to-do:** https://basecamp.com/1810084/projects/18117394/todos/441951114

**Comment by remcotolsma** [comment]
Request/tip from customer:

> Het zou mooi zijn als je in de statistieken ook een rapportje of overzicht hebt qua aantal / hoeveelheid in euro's terugkerende betalingen je hebt tov een enkele betaling.
> 
> Nu zie ik bv bij ons dat we 139 succesvolle betalingen hebben, maar als ik het terugkerende aantal daarvan wil weten, moet ik alle betalingen langs lopen of dat blauwe icoontje aan staat.

_Internal HelpScout ticket:_ https://secure.helpscout.net/conversation/1618113417/22642?folderId=1425710

**Comment by remcotolsma** [comment]
Question from customer:

> Is er een manier om vanuit de plugin, onder de kop 'Rapporten', terug te kijken naar de voorgaande jaren. Nu wordt rapportage automatisch ingesteld op het huidige jaar.

Answer from Pronamic:

> Het is helaas momenteel niet mogelijk om terug te kijken naar een voorgaand jaar. We hebben wel het idee om een rapportage add-on te ontwikkelen die meer inzicht geeft in de betalingen. Vergelijkbaar met bijvoorbeeld de nieuwe WooCommerce Analytics module: https://woocommerce.com/document/woocommerce-analytics/.

**Comment by remcotolsma** [comment]
We had the following idea:

> A calendar overview with all follow-up payments for the upcoming year/month

- https://github.com/pronamic/wp-pronamic-pay/issues/288

Suggestion from @rvdsteege to include this in a reporting add-on:

> I would add something like this to either a Subscriptions or Reporting add-on and leave this out for now.

https://github.com/pronamic/wp-pronamic-pay/issues/288#issuecomment-1024119246

**Comment by rvdsteege** [comment]
From customer:

> Aangezien het nieuwe jaar is begonnen zijn de betalings overzicht van mijn reporten onder de pronamic dasboard weg. Ik zou heel graag onder reporten nog mijn inkomsten willen zien van 2021, ik kan nu alleen de betalingen van 2022 zien. Ik zou graag beiden jaren tegen elkaar kunnen vergelijken.

_Internal Help Scout ticket: https://secure.helpscout.net/conversation/1803258090/23607_

**Comment by rvdsteege** [comment]
From customer using _Pronamic Pay with Mollie for Gravity Forms_ plugin:

> In the stats or reports it will be good to see total revenue generated. Per month & till date. For eg see his woocomerce stats. I am hope this must be in your roadmap
> 
> ![WooCommerce stats](https://github.com/pronamic/wp-pronamic-pay/assets/10371164/a22cfcd3-1aec-492d-ae1e-d46fca3b054d)

https://wordpress.org/support/topic/total-revenue-stats/

My reply:

> A dedicated add-on for stats has been on our 'wish list' for some time. That will probably also allow us to update the dashboard widget with detailed stats.

**Comment by rvdsteege** [comment]
From https://wordpress.org/support/topic/last-year-reports/

> Is it possible to see last year salse report?

My reply:

> Unfortunately, the current reporting functionality is limited to this year’s data. Enhanced reporting capabilities are on our radar, but changes in this area are not expected in the short term.
